content/_partials/scriptlinks.html.haml

```diff
@@ -1,23 +1,23 @@
-%link{:href => '/css/font-awesome.min.css',
+%link{:href => "#{site.base_url}/css/font-awesome.min.css",
```

Comment:
I don't use `./gradlew run`, never have, and have been adamant about not supporting it ever since KK added it.

This would cause those links to be incorrect, since the default `site.base_url` is `localhost:4242`, but that can be easily overridden with an [Awestruct profile](http://awestruct.org/profiles/) redefining `site.base_url`.


I would just as soon remove that `run` task from the `build.gradle` than hack the world around it to make it work.